src/index.html

```diff
@@ -3,11 +3,444 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Creative bakery</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+     rel="icon"
+     type="image/png"
+     href="images/icon/icon-cookie.png"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+        <div class="top--bar">
+          <a href="#" class="top--bar__logo-link">
+            <img
+              src="images/top-bar__logo.png"
+              alt="Creative__Bakery"
+              class="top--bar__logo"
+            >
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item nav__item--1">
+                <a
+                  href="#product"
+                  class="nav__link"
+                >
```

Comment:
Transferring attributes should be done only when there are 3 or more attributes, need to check and fix it everywhere
```suggestion
                <a href="#product" class="nav__link">
```